cli/src/command/account.rs

```diff
@@ -926,40 +926,62 @@ async fn print_address(account: &Account, address: &AccountAddress) -> Result<()
     let addresses = account.addresses_with_unspent_outputs().await?;
     let current_time = iota_sdk::utils::unix_timestamp_now().as_secs() as u32;
 
+    let mut output_ids = Vec::new();
+    let mut address_amount = 0;
+    let mut address_nts = NativeTokensBuilder::new();
+    let mut address_nfts = Vec::new();
+    let mut address_aliases = Vec::new();
+    let mut address_foundries = Vec::new();
```

Comment:
Could also remove the `address_` prefix, but I don't really care :trollface:

Reply:
Well let's remove if it annoys the both of you? Also annoys me.